src/devices/schneider_electric.ts

```diff
@@ -1340,6 +1522,93 @@ const definitions: Definition[] = [
             commandsOnOff({'endpointNames': ['switch_sw']}),
         ],
     },
+    {
+        zigbeeModel: ['E8331DST300ZB'],
+        model: 'E8331DST300ZB',
+        vendor: 'Schneider Electric',
+        description: 'Wiser AvatarOn 2G Dimmer Switch',
```

Comment:
```suggestion
        description: 'Wiser AvatarOn 2G dimmer switch',
```

(same for other `description`s)